### PR TITLE
Adding Update Bootstrap Command for GCP

### PR DESCRIPTION
**Pull request by mmcelroy91**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-88884
Doc ticket https://jira.d2iq.com/browse/D2IQ-90120

## Description of changes being made

Add functionality to update bootstrap credentials (similar to aws and azure)

`dkp update bootstrap credentials gcp`

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4497.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by mmcelroy91** [comment]
PLEASE NOTE this is a work in progress since GCP is now.  Not all CLI and other steps are correct yet... adding them as we go.  Please feel free to comment and add CLI and other steps after testing is completed on each.

**Review by supershal — APPROVED** [review]
lgtm

**Comment by msdolbey** [comment]
@mmcelroy91  - added some changes - mostly linting and saw a broken link - if you have updates, you'll have to pull. But, other than that - looks good to me!

File: pages/dkp/konvoy/2.3/choose-infrastructure/gcp/advanced/new/index.md [hunk 1]
```diff
@@ -13,36 +13,59 @@ enterprise: false
...
 
 1.  Verify the API server is up (the Nodes will not be ready until CSI is deployed):
 
-    ```
+    ```bash
     kubectl get nodes
-    ```
\ No newline at end of file
+    ```
+
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.3/choose-infrastructure/gcp/advanced/new/index.md:67]: Here too, maybe we could get some output?

**Review by gvasquezvargas — APPROVED** [review]
Looks very good to me! Since this is wip, maybe we can ask dev to provide some outputs, specially in the parts where we as customers to "verify" or "confirm" something?

File: pages/dkp/konvoy/2.3/choose-infrastructure/gcp/advanced/new/index.md (hunk 1 above)
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.3/choose-infrastructure/gcp/advanced/new/index.md:67]: Agree!  I guess I could go ahead and ask for that now.

**Pull request merged**